x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/types.ts

```diff
@@ -7,6 +7,7 @@
 
 export interface CspmUsage {
   indices: CspmIndicesStats;
+  accounts: CspmAccountsStats[] | [];
```

Comment:
```suggestion
  accounts: CspmAccountsStats[];
```

can you clarify this?